### PR TITLE
[BugFix] add verify logit_bias to avoid crash because of IndexError 

### PR DESCRIPTION
**Pull request by ehuaa**
## Motivation

fix this issue https://github.com/sgl-project/sglang/issues/7670.
## Modifications

Add check of logit_bias in verify to avoid IndexError in https://github.com/sgl-project/sglang/issues/7670, which may cause server crash. 
Ensure the keys in logit_bias is in [0, vocab_size - 1]
## Checklist

- [ ] Format your code according to the [Code Formatting with Pre-Commit](https://docs.sglang.ai/references/contribution_guide.html#code-formatting-with-pre-commit).
- [ ] Add unit tests as outlined in the [Running Unit Tests](https://docs.sglang.ai/references/contribution_guide.html#running-unit-tests-adding-to-ci).
- [ ] Update documentation / docstrings / example tutorials as needed, according to [Writing Documentation](https://docs.sglang.ai/references/contribution_guide.html#writing-documentation-running-docs-ci).
- [ ] Provide throughput / latency benchmark results and accuracy evaluation results as needed, according to [Benchmark and Profiling](https://docs.sglang.ai/references/benchmark_and_profiling.html) and [Accuracy Results](https://docs.sglang.ai/references/accuracy_evaluation.html).
- [ ] For reviewers: If you haven't made any contributions to this PR and are only assisting with merging the main branch, please remove yourself as a co-author when merging the PR.
- [ ] Please feel free to join our Slack channel at https://slack.sglang.ai to discuss your PR.

### CODE REVIEW

File: python/sglang/srt/sampling/sampling_params.py [hunk 1]
```diff
@@ -131,6 +131,13 @@ def verify(self):
                     f"min_new_tokens must be in [0, max_new_tokens({self.max_new_tokens})], got "
                     f"{self.min_new_tokens}."
                 )
+        if self.logit_bias is not None:
+            for token_id in self.logit_bias:
+                if not 0 <= int(token_id) < vocab_size:
+                    raise ValueError(
+                        f"logits_bias must has keys in [0, {vocab_size - 1}], got "
+                        f"{token_id}."
+                    )
```
[the remark this answers is not in the document]
Comment (ehuaa) [reply python/sglang/srt/sampling/sampling_params.py]: 1.if a key in logit_bias is not a string representation of an integer, it will be found in previous check and send client a bad request error of "invalid literal for int() with base 10: 'abc'".
2. I will correct this typo now.

**Comment by RenaultAI** [comment]
Thanks for the quick turnaround!

**Review by hnyls2002 — APPROVED** [review]
